prow/gerrit/adapter/adapter.go

```diff
@@ -238,9 +238,7 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 				latestReportTime = message.Date.Time
 			}
 		}
-		if latestReport == nil {
-			logger.Warnf("Found nil latest report")
-		} else {
+		if latestReport != nil {
```

Comment:
If latestReport == nil and its been multiple hours since we created the prowjob, that might indicate something to warn about?

Reply:
We can have no always_run jobs in a repo, and user can manually trigger jobs, so I would not make this assumption.